src/test/java/core/basesyntax/strategy/AddOperationHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.OperationType;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddOperationHandlerTest {
+    private static OperationHandler handler;
+    private static Transaction transaction;
+
+    @BeforeClass
+    public static void beforeClass() {
+        handler = new AddOperationHandler();
+        Storage.getFruits().put(new Fruit("apple"), 10);
+    }
+
+    @AfterClass
+    public static void afterClass() {
```

Comment:
should we clear storage after each test or after all tests?

Reply:
Changed it